.github/workflows/build.yml

```diff
@@ -0,0 +1,163 @@
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+#
+# SPDX-License-Identifier: MIT
+
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false #true
+      matrix:
+        os:
+          - ubuntu-22.04
+          - windows-2022
+          - macos-12
+        build_type:
+          - Debug
+          #- Release
+        client_only:
+          - client_only
+          - client_and_ui
+        qt:
+          - version: "5.15.2"
+            requested: "5.15"
+          - version: "6.5.3"
+            requested: "6.5.*"
+            modules: qtshadertools
+
+    steps:
+      - name: Install Qt with options and default aqtversion
+        uses: jurplel/install-qt-action@v3
+        with:
+          aqtversion: null # use whatever the default is
```

Comment:
I'm trying to understand why this is here.

If it's not specified _at all_, the default will be used, right? That's why it's the default. Why would it be necessary to explicitly **ask** for a default?

Reply:
We can remove ofcourse, but I feel like this might be needed in future, also helps one knowing that it uses aqt behind the scenes

Reply:
That's fair.